open_schools_platform/common/services.py

```diff
@@ -96,3 +96,7 @@ def get_object_by_id_in_field_with_checks(filters, request, fields: Dict[str, Ca
             result.append(None)
 
     return result
+
+
+def filter_list_from_empty_strings(lst: list):
+    return [item for item in lst if item]
```

Comment:
i think it is standart python func
if it isn't locate it to utils

Reply:
Yes, I think we can use lambda and filter() here.